test/parallel/sdk.API.batch.test.ts

```diff
@@ -0,0 +1,292 @@
+/*
+ *
+ * @group sdk
+ * @group parallel
```

Comment:
Are we running both groups? perhaps we are running this test twice?

Reply:
Not all our SDK tests are covered by parallel group and I supposed that we need to check either only parallel tests or only SDK ones. I can remove one of the groups (I think it would be parallel).

Reply:
remove one plz otherwise, we are running those tests twice.

Reply:
done